package.json

```diff
@@ -66,6 +66,7 @@
     "postcss-nested": "^5.0.5",
     "preval.macro": "^5.0.0",
     "prop-types": "^15.7.2",
+    "query-string": "^7.0.1",
```

Comment:
As per rule https://www.ssw.com.au/rules/do-you-keep-your-npm-packages-up-to-date
When a package is added, we usually update all other packages

Reply:
See #740